src/test/java/org/assertj/core/internal/iterables/Iterables_assertContainsExactly_Test.java

```diff
@@ -68,19 +100,19 @@ public void should_fail_if_array_of_values_to_look_for_is_empty_and_actual_is_no
   @Test
   public void should_throw_error_if_array_of_values_to_look_for_is_null() {
     assertThatNullPointerException().isThrownBy(() -> iterables.assertContainsExactly(someInfo(), emptyList(), null))
-                                    .withMessage(valuesToLookForIsNull());
+      .withMessage(valuesToLookForIsNull());
```

Comment:
general comment: use assertj code formatting settings, the changes in this PR is 90% code formatting which is a lot of noise. thanks!

Reply:
Sorry, I didn't notice my IDE had changed these. I'd even made sure the 'reformat' option was not selected. Will try to undo.